### PR TITLE
sci-biology/arb: Version bump to 6.0.6

### PR DESCRIPTION
**Pull request by Peter-Levine**
Bug: https://bugs.gentoo.org/637118
Package-Manager: Portage-2.3.10, Repoman-2.3.3

### CODE REVIEW

File: sci-biology/arb/arb-6.0.6.ebuild [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-6.0.6-glapi.patch
+	"${FILESDIR}"/${PN}-6.0.6-tc-flags.patch
+	"${FILESDIR}"/${PN}-6.0.6-arb_install.patch
+)
+
+src_unpack() {
+	default
+	mv arbsrc* ${P}
```
Comment (monsieurp) [inline sci-biology/arb/arb-6.0.6.ebuild]: missing `die ||`. There are missing dies all over the place. Could you please add them? Thanks.

File: sci-biology/arb/arb-6.0.6.ebuild [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+}
+
+src_prepare() {
+	default
+
+	cp config.makefile.template config.makefile
+	mkdir "${S}"/patches.arb   # Test script expects ${ARBHOME}/patches.arb to exist
+
+	use amd64 && sed -i -e 's@ARB_64 := 0@ARB_64 := 1@' config.makefile
```
Comment (monsieurp) [inline sci-biology/arb/arb-6.0.6.ebuild]: Please remove that line and use a much simpler if clause along with a missing `|| die`.

File: sci-biology/arb/arb-6.0.6.ebuild (hunk 1 above)
Comment (Peter-Levine) [reply sci-biology/arb/arb-6.0.6.ebuild]: No problem.  I was just using `arb-5.3.ebuild` as a template but I'll clean it up.

File: sci-biology/arb/arb-6.0.6.ebuild [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+	if use amd64; then
+		mv arb.tgz arb.64.gentoo.tgz || die
+	fi
+	if use x86; then
+		mv arb.tgz arb.32.gentoo.tgz || die
+	fi
+	ln -s arb.*.tgz arb.tgz || die
+	ARBHOME="${D}/opt/arb" "${S}/arb_install.sh" || die
+
+	cat <<- EOF > "${S}/99${PN}" || die
```
Comment (SoapGentoo) [inline sci-biology/arb/arb-6.0.6.ebuild]: FYI, with `<<-` you can indent the stuff you're piping (makes it slightly easier to read)

**Pull request closed without merging**